tests/testutils/data/m/minimal_messages_config.py

```diff
@@ -0,0 +1,5 @@
+"""Dummy test file to test the --minimal-messages-config option."""
```

Comment:
~~I don't understand how exactly it's testing ``minimal-messages-config``, there's supposed to be other message there but as we did not disable them so they should be raised if the option did not work ?~~

```suggestion
"""
Dummy test file to test the --minimal-messages-config option.

This file is tested in ``test_minimal_messages_config_enabled`` and
``test_minimal_messages_config_excluded_file``.
"""
```